Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -827,31 +827,31 @@ - (void)dealloc
 
 @implementation WebUITextIndicatorData (WebUITextIndicatorInternal)
 
-- (WebUITextIndicatorData *)initWithImage:(CGImageRef)image textIndicatorData:(const WebCore::TextIndicatorData&)indicatorData scale:(CGFloat)scale
+- (WebUITextIndicatorData *)initWithImage:(CGImageRef)image textIndicator:(const WebCore::TextIndicator&)indicator scale:(CGFloat)scale
 {
     if (!(self = [super init]))
         return nil;
 
     _dataInteractionImage = [PAL::allocUIImageInstance() initWithCGImage:image scale:scale orientation:UIImageOrientationDownMirrored];
-    _selectionRectInRootViewCoordinates = indicatorData.selectionRectInRootViewCoordinates;
-    _textBoundingRectInRootViewCoordinates = indicatorData.textBoundingRectInRootViewCoordinates;
-    _textRectsInBoundingRectCoordinates = createNSArray(indicatorData.textRectsInBoundingRectCoordinates).leakRef();
-    _contentImageScaleFactor = indicatorData.contentImageScaleFactor;
-    if (indicatorData.contentImageWithHighlight)
-        _contentImageWithHighlight = [PAL::allocUIImageInstance() initWithCGImage:indicatorData.contentImageWithHighlight.get()->nativeImage()->platformImage().get() scale:scale orientation:UIImageOrientationDownMirrored];
-    if (indicatorData.contentImage)
-        _contentImage = [PAL::allocUIImageInstance() initWithCGImage:indicatorData.contentImage.get()->nativeImage()->platformImage().get() scale:scale orientation:UIImageOrientationUp];
-
-    if (indicatorData.contentImageWithoutSelection) {
-        auto nativeImage = indicatorData.contentImageWithoutSelection.get()->nativeImage();
+    _selectionRectInRootViewCoordinates = indicator->selectionRectInRootViewCoordinates();
+    _textBoundingRectInRootViewCoordinates = indicator->textBoundingRectInRootViewCoordinates();
+    _textRectsInBoundingRectCoordinates = createNSArray(indicator->textRectsInBoundingRectCoordinates()).leakRef();
+    _contentImageScaleFactor = indicator->contentImageScaleFactor();
+    if (indicator->contentImageWithHighlight())
+        _contentImageWithHighlight = [PAL::allocUIImageInstance() initWithCGImage:indicator->contentImageWithHighlight()->nativeImage()->platformImage().get() scale:scale orientation:UIImageOrientationDownMirrored];
+    if (indicator->contentImage())
+        _contentImage = [PAL::allocUIImageInstance() initWithCGImage:indicator->contentImage()->nativeImage()->platformImage().get() scale:scale orientation:UIImageOrientationUp];
+
+    if (indicatorData->contentImageWithoutSelection()) {
+        auto nativeImage = indicator->contentImageWithoutSelection()->nativeImage();
```

Comment:
These would all require `indicator.` instead of `indicator->`, right? Since it's a reference